streamx-plugin/streamx-flink-packer/src/main/scala/com/streamxhub/streamx/flink/packer/docker/DockerRetriever.scala

```diff
@@ -46,9 +47,9 @@ object DockerRetriever {
   lazy val dockerHttpClientBuilder: ApacheDockerHttpClient.Builder = new ApacheDockerHttpClient.Builder()
     .dockerHost(dockerClientConf.getDockerHost)
     .sslConfig(dockerClientConf.getSSLConfig)
-    .maxConnections(100)
-    .connectionTimeout(Duration.ofSeconds(30))
-    .responseTimeout(Duration.ofSeconds(45))
+    .maxConnections(ConfigHub.get(CommonConfig.DOCKER_MAX_CONNECTIONS))
```

Comment:
The value obtained here is not associated with the value set by the user in application.yml.It should be, first get the value set by the user from applicacy.yml, if not set, take the default value